source/taskmanager-client/src/main/java/nl/aerius/taskmanager/client/BrokerConnectionFactory.java

```diff
@@ -146,10 +146,19 @@ public void deRegisterClient(final TaskManagerClientSender taskManagerClient) {
     taskManagerClients.remove(taskManagerClient);
   }
 
+  /**
+   * @deprecated Use {@link #close()}
+   */
+  @Deprecated
+  public void shutdown() throws Exception {
```

Comment:
Does it need to throw Exception?

Reply:
No it doesn't. Not sure how that got added...